index.html

```diff
@@ -4,43 +4,193 @@
+                        <img src="../img/Counter.png">
+                        &nbsp;
+                        <span>Back End Dev</span>
+                    </li> 
+                    &nbsp;
+                    <li class ="list-element">
+                        <img src="../img/Counter.png">
+                        &nbsp;
+                        <span>2015</span>
+                    </li> 
+                </p>
+            </div>
+            <div class ="flex-item flex-item-4">
+                <p class="project-content-4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class ="flex-item flex-item-5">
+                <img class="project-content-5" src="../img/html-css-js.png">
+            </div>
+            <div class ="flex-item flex-item-6"><button>See Project</button></div>
+
+        </div>
+
+        <div class="grid-item flex-container">
+            <div class ="flex-item flex-item-1"><img class ="project-content-1" src="../img/Project-4.png"></div>
+            <div class ="flex-item flex-item-2"><h1 class="project-content-2">Multi-Post Stories</h1></div>
+            <div class ="flex-item flex-item-3">
+                <ul class ="project-content-3"> 
+                    <li class ="title-element">CANOPY</li> 
+                    &nbsp;
+                    <li class ="list-element">
+                        <img src="../img/Counter.png">
+                        &nbsp;
+                        <span>Back End Dev</span>
+                    </li> 
+                    &nbsp;
+                    <li class ="list-element">
+                        <img src="../img/Counter.png">
+                        &nbsp;
+                        <span>2015</span>
+                    </li> 
+                </p>
+            </div>
+            <div class ="flex-item flex-item-4">
+                <p class="project-content-4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class ="flex-item flex-item-5">
+                <img class="project-content-5" src="../img/html-css-js.png">
+            </div>
+            <div class ="flex-item flex-item-6"><button>See Project</button></div>
+        </div>
+
+
+
+    <div class="grid-item flex-container">
+        <div class ="flex-item flex-item-1"><h1 class="about-title">About <br> Myself</h1></div>
+        <div class ="flex-item flex-item-2">
+            <p class ="about-paragraph">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.   
+            </p> 
+        </div>
+        <div class ="flex-item flex-item-3">
+            <div class="about-connect">
                 LET’S CONNECT
             </div>
-            <br>
```

Comment:
- [ ] The word " LET’S CONNECT" should use a header tag, e.g `h2`. Kindly replace this `div` with it.